odfuzz/config.py

```diff
@@ -2,7 +2,21 @@
 
 import os
 
-from odfuzz.constants import ENV_ODFUZZ_CERTIFICATE_PATH, DEFAULT_ASYNC_REQUESTS_NUM, DEFAULT_SAP_CLIENT, DEFAULT_DATA_FORMAT, DEFAULT_URLS_PER_PROPERTY, ENV_SAP_CLIENT, ENV_DATA_FORMAT, ENV_URLS_PER_PROPERTY, ENV_ASYNC_REQUESTS_NUM
+from odfuzz.constants import (
+    DEFAULT_ASYNC_REQUESTS_NUM,
+    DEFAULT_DATA_FORMAT,
+    DEFAULT_ENCODE_STRING,
+    DEFAULT_SAP_CLIENT,
+    DEFAULT_URLS_PER_PROPERTY,
+    ENV_ASYNC_REQUESTS_NUM,
+    ENV_DATA_FORMAT,
+    ENV_ENCODE_STRING,
+    ENV_ODFUZZ_CERTIFICATE_PATH,
+    ENV_SAP_CLIENT,
+    ENV_URLS_PER_PROPERTY,
+)
+from odfuzz.utils import encode_double, encode_string
```

Comment:
I don't understand why this should be in Config and later called this way from generators. 
seems unnecessary complicated, when usually the config just stores the values grabbed from CLI or env variable and thats all he does.. all logic based on the value of such variable in place where is actually used.